protos/farm_ng/core/pose.proto

```diff
@@ -26,12 +26,11 @@ message Pose {
   string frame_a = 2;
   string frame_b = 3;
 
+  // linear velocity of body b (with respect to body b) that exists in frame a.
```

Comment:
From convo w/ @strasdat 
Change comment to something more like:
// Linear velocity of frame b with respect to frame a
// Egocentric velocity of body b with regard to frame a.
// e.g., if b is 'robot' and 'a' is world,this is the linear velocity of the robot (x is moving forward, y left, z up)
// If 'b' is a tool and 'a' is the robot, the robot velocity is not included. Only the actuation of the tool relative to robot's body, but in the tool's axes.

Maybe change name to `linvelv_of_b_in_b`. There are really 3 reference frames, so the name is hard to nail down here